cracked_hash_stats.py

```diff
@@ -122,14 +124,11 @@ def helpmsg():
 showPopularPasswords = True  # List top x most popular passwords
 popularPasswordCount = 15
 ignoreHistory0 = True  # Ignore history0 entries *_history0 hashes
-showModernStats = False  # Show a stats block for current passwords
+showModernStats = True  # Show a stats block for current passwords
 showHistoryStats = False  # Show a stats block for history passwords
 showCombinedStats = False  # Combine stats of both modern and history passwords
 showUncracked = False  # Print usernames with uncracked passwords
-
-if len(sys.argv) < 4:  # If no options are specified, use default options
-    showModernStats = True
-    showHistoryStats = True
+ignoreBlankPWUsers = True  # Ignore users whose hash == blank password
 
 try:
     opts, args = getopt.getopt(sys.argv[1:], 'hm:pc:CMHu',
```

Comment:
if you are adding options then you need to add them to the getopt.getopt(sys.argv[1:], .... section.

if the option does not have an argument then dont add a colon or = sign.